jest.config.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+	testMatch: [
+		"<rootDir>/src/**/__tests__/**/*.{js,jsx}",
+		"<rootDir>/src/**/*.{spec,test}.{js,jsx}"
+	],
+	transform: {
+		'^.+\\.(js|jsx|ts|tsx)$': '<rootDir>/node_modules/babel-jest',
```

Comment:
Prefer using `ts-jest` for typescript files for more accurate coverage data.

https://github.com/facebook/jest/issues/7804

Reply:
Removing `ts` and `tsx` for now, since we are not support it yet